sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4

```diff
@@ -135,7 +137,8 @@ statement
         (ALTER | CHANGE) COLUMN? qualifiedName
         (TYPE dataType)? (COMMENT comment=STRING)? colPosition?        #alterTableColumn
     | ALTER TABLE tableIdentifier partitionSpec?
-        CHANGE COLUMN? identifier colType colPosition?                 #changeColumn
+        CHANGE COLUMN?
+        colName=errorCapturingIdentifier colType colPosition?          #changeColumn
```

Comment:
@yeshengm . Since we remove the test coverage for this and this PR doesn't support this, this PR should not touch this line.
> This is due to a recent PR on qualified column name. Fixed by removing error capture in that syntax.

Could you revert this?